src/test/clojure/clara/test_rules.clj

```diff
@@ -27,7 +29,21 @@
            LinkedList
            ArrayList]))
 
-(use-fixtures :once schema.test/validate-schemas)
+(defn opts-fixture
+  ;; For operations other than replace-facts uc/get-ordered-update-cache is currently
```

Comment:
Should you instead run all tests here twice, one for each backing cache? A :once fixture makes that easy enough.